Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: all integration-test deployer geth-l2 batch-submitter data-transport-layer test
+
+all:
+	./build-local.sh
+
+docker:
+	./docker/build.sh
+
+integration-tests:
+	./build-local.sh -s integration_tests
+
+deployer:
+	./build-local.sh -s deployer
+
+geth-l2:
+	./build-local.sh -s geth_l2
+
+batch-submitter:
+	./build-local.sh -s batch_submitter
+
+data-transport-layer:
+	./build-local.sh -s data_transport_layer
+
+up: 
+	./up.sh
+
+# make up <package-name>
+up %:
+	./up.sh -l $@
```

Comment:
this SEEMS to work but might be messing each other up

Reply:
will update the command here 

https://github.com/ethereum-optimism/optimism-integration/pull/84/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R115 

if it works